stripe.go

```diff
@@ -1278,6 +1278,7 @@ type requestTelemetry struct {
 var appInfo *AppInfo
 var backends Backends
 var encodedStripeUserAgent string
+var encodedStripeUserAgentReady *sync.Once
```

Comment:
Side-by code review :). This doesn't need to be a pointer. It can directly use `encodedStripeUserAgentReady sync.Once` and won't need initialization.

Reply:
We would still need initialization to support multiple `SetAppInfo` calls. You are right that the pointer isn't required.


Thank you for the review :)